R/pmcmc.R

```diff
@@ -99,8 +100,7 @@ pmcmc <- function(data,
                                   'start_date' = 0),
                   pars_max = list('beta' = 1, 
                                   'start_date' = 1e6),
-                  pars_sd = list('beta' = 0.001, 
-                                 'start_date' = 0.5),
+                  cov_mat = c('beta' = 0.001^2, 'start_date' = 0.5^2),
```

Comment:
I think a form which shows this as a matrix (even with zeros) would be easier to modify for a user.
```
cov_mat = matrix(c(0.001^2, 0,
                     0, 0.5^2), 
```
would be equivalent?